.github/workflows/build_template.yml

```diff
@@ -23,7 +23,8 @@ jobs:
           - name: Check generated files are up to date
             working-directory: ${{ inputs.modulepath }}
             run: |
-              go generate -x ./...
+              make generate
```

Comment:
I think this will fail when being called by at least the following workflows (knowing that `main_template` forwards to `build_template`):
- https://github.com/gnolang/gno/blob/master/.github/workflows/contribs.yml
- https://github.com/gnolang/gno/blob/master/.github/workflows/misc.yml
- https://github.com/gnolang/gno/blob/master/.github/workflows/gnoland.yml

Reply:
I really hate that we enforce random makefile directives within subfolders, just to follow a structure (prime example is `make lint`, `make test` etc)